docs/content/docs/deployment/security/updates.md

```diff
@@ -0,0 +1,73 @@
+---
```

Comment:
That's decided on a case-by-case basis.

Generally speaking we aim to publish vulnerabilities once a mitigation is in place (either a new release or patch, depending on the severity).
In the case of Log4j we communicate it right away because the cat was out of the bag already at that point.

Reply:
Thanks for the explanation! I agree that Log4j was really an exception many ways.